src/compiler/core.ts

```diff
@@ -394,6 +394,14 @@ namespace ts {
         return result;
     }
 
+    export function mapIter<T, U>(iter: Iterator<T>, mapFn: (x: T) => U): Iterator<U> {

```

Comment:
I'll suggest not abbreviating. Instead of mapIter -> mapIterator.

Same for the parameter: iter -> iterator